h/panels/navbar.py

```diff
@@ -7,6 +7,7 @@
 from pyramid_layout.panel import panel_config
 
 from h.i18n import TranslationString as _  # noqa
+from h.services.profile_group import profile_groups_factory
```

Comment:
Hi @hmstepanek !

Oh, dear, I didn't know you were going to make use of services here (until this point AFAIK the web views have iterated directly over `user.groups`). Unfortunately, `ProfileGroupService` is going away imminently in favor of the new and more consistent `ListGroupsService`. Also, unfortunately all of this related work is terribly in-flight right now (some has already landed on `master` while there is yet an open PR at #4805 and another one touching `session` teed up after that).

The good news is that the update needed here will probably be mostly confined to importing a different service and calling a method on it. And I can certainly help you out once things are in place. I hope that all of the pieces of `ListGroupsService`, not to mention an updated version of the `session` module (with a more consistent treatment of `request.user.groups`) should land tomorrow. Sorry I didn't spot this earlier, but I didn't anticipate you'd use services here (I mean, services is a better path in the long run for sure, it just hasn't been the pattern used here until now, I don't think?).

Reply:
Ah, ok I think I got confused when I double checked against the git hist and thought it was the other way around (profile_groups was staying). I'll change it to use /groups instead... I don't anticipate this getting committed until after all the groups work that is in-flux right now is completed and we have a semi-stable version of the api. :) And yes, you are correct, we were not using a service in this code before now.